.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Setup Java 8
+      - name: Setup Java
```

Comment:
Do we care to specify the version in the `Setup Java` step? The answer may be no - more just thinking of me trying to remember where the java version is specified for the SDK and and always forgetting. 🙈

Reply:
This is the version of Java that's use to build the modules and run the tests - we should try and keep it in line with the version we compile for, but it was actually already bumped to Java 11 in CI 😆  (this is just removing the "8" from the name of the task since it's incorrect)

Reply:
LOL - here I was assuming it was right the whole time 🙃